### DISCUSSION TITLE
The temporary folder option is moved to project

### DISCUSSION THREAD

**Discussion by gkalemis** [staxrip/staxrip#1320] [General]
It is my understanding that the option to remove the temporary files after the encoding is moved from general settings to project settings. 
I was wondering why as now you have to make the same option if you don't want the default into any of your templates.
Thank you,

**Comment by Dendraspis** [comment]
Not sure if I understand your point....

It was a general setting and now it is a project based one including a lot more settings, giving you the possibility to set different settings for different projects/templates.

>  you have to make the same option if you don't want the default into any of your templates.

In case you always want to have the temp files deleted, set it and save the template.  🤔

**Reply by gkalemis** [reply]
Don't get me wrong. I was just wondering. 
Thank you very much for all the efforts and take care.
